<?xml version="1.0"?>

<launch>

    <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

    <!-- MiFlora Service Node -->
    <node pkg="miflora_server" type="sensor_server.py" name="sensor_server" 
          clear_params="true" output="screen" />

</launch>